spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -249,6 +249,22 @@
         expect(response.success?).to eq(false)
       end
     end
+
+    context 'when doc_auth_read_additional_pii_attributes_enabled is enabled' do
+      let(:success_response_body) { LexisNexisFixtures.true_id_response_success }
+
+      it 'reads the additional PII attributes' do
+        allow(IdentityConfig.store).to receive(:doc_auth_read_additional_pii_attributes_enabled).
+          and_return(true)
+
+        pii_from_doc = response.pii_from_doc
+
+        expect(pii_from_doc.first_name).to eq('LICENSE')
```

Comment:
Should the first name be LICENSE?

Reply:
That is read from here: https://github.com/18F/identity-idp/blob/3384b65c238fe17ecc237b2fe3790aea5c127e54/spec/fixtures/proofing/lexis_nexis/true_id/true_id_response_success.json#L538-L546

I switched to this fixture for this test since it includes these additional attributes.